src/Statistics.jl

```diff
@@ -1,21 +1,41 @@
 using StatsBase: AbstractWeights, Weights, ProbabilityWeights, values, varcorrection
+
+_mean_weights(n::Int) = Weights(ones(n), n)
+_var_weights(n::Int) = ProbabilityWeights(ones(n), n)
+
 @doc doc"""
-    mean(M, x, weights=Weights(ones(n)); x0=x[1], stop_iter=100, kwargs... )
+    mean(M::Manifold, x::AbstractVector; kwargs...)
+    mean(M::Manifold, x::AbstractVector, w::AbstractWeights; kwargs...)
 
-computes the Riemannian center of mass also known as Karcher mean of the vector
-`x` of `n` points on the [`Manifold`](@ref) `M`. This function
-uses the gradient descent scheme. Optionally one can provide
-weights $w_i$ for the weighted Riemannian center of mass.
-The general formula to compute the minimizer reads
+Compute the (optionally weighted) Riemannian center of mass also known as
+Karcher mean of the vector `x` of points on the [`Manifold`](@ref) `M`, defined
+as the point that satisfies the minimizer
 ````math
-\argmin_{y\in\mathcal M} \frac{1}{2}\sum_{i=1}^n w_i\mathrm{d}_{\mathcal M}^2(y,x_i),
+\argmin_{y\in\mathcal M} \frac{1}{2 \sum w}\sum_{i=1}^n w_i\mathrm{d}_{\mathcal M}^2(y,x_i),
```

Comment:
Do you mean `\sum_i w_i`?

Reply:
Yes. I was using the abbreviated notation StatsBase uses for weight summation, but I can expand it.